index.html

```diff
@@ -0,0 +1,201 @@
+                </div>
+                <div class = "shared-part">
+                    <h2>Dark Patterns At Scale</h2>
+                    <p>apr 5, 2019 —Lorem ipsum dolor, sit amet consectetur adipisicing elit. Modi reprehenderit totam quam! Expedita molestiae, totam voluptas hic esse vel? Maxime, fuga nostrum distinctio unde corporis ex culpa non voluptatibus laboriosam.</p>
+                </div>
+                <div class = "shared-part">
+                    <h2>The Simplest Way To Load CSS Asynchronously</h2>
+                    <p>apr 5, 2019 —Lorem ipsum dolor sit amet consectetur adipisicing elit. Hic expedita dignissimos odio, sapiente dolorum sunt deserunt quaerat! Ex iste molestias qui enim aliquam explicabo, ut, voluptates eos mollitia amet quibusdam.</p>
+                </div>
+                <div class = "shared-part">
+                    <h2>Web platform news</h2>
+                    <p>apr 5, 2019 —Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consequatur eligendi, numquam explicabo eos illum fugiat impedit distinctio aliquid blanditiis. Nam magnam aliquid consequuntur odit dolorum iusto veniam sint saepe facilis.</p>
+                </div>
+            </section>
+
+            <section class = "design">
+                <h2>Design Systems</h2>
+                <p>Meet Design Systems, a recent Smashing book in which Alla Kholmatova explores how to set up an effective design system to create great digital products. With common traps, gotchas and lessons learned.</p>
+            </section>
+        
+            <section class = "latest-posts" id = "guides">
+                
+                <div class = "latest-posts-part">
+                    <h2>New Front-End Challenges To Master</h2>
+                </div>
+                <div class = "latest-posts-part">
+                    <h2>Performance Optimization & Strategy </h2>
+                </div>
+                <div class = "latest-posts-part">
+                    <h2>Improving Workflow & Data Protection </h2>
+                </div>
+                <div class = "latest-posts-part">
+                    <h2>Design Patterns & Checklists</h2>
+                </div>
+                <div class = "latest-posts-part">
+                    <h2>Dark Patterns At Scale</h2>
+                </div>
+                <div class = "latest-posts-part">
+                    <h2>The Simplest Way To Load CSS Asynchronously</h2>
+                </div>
+                <div class = "latest-posts-part" id = "newsletter">
+                    <h2>Smashing Newsletter</h2>
+                </div>
+            </section>
+            <footer>
+                <p>BROWSE ALL SMASHING MAGAZINE TOPICS</p>
+                <div class="bottom-menu">
+                    <ul>
```

Comment:
The li items inside the `<ul>` tags should be positioned using the float or flex display property.